data/json/terrain.json

```diff
@@ -5203,6 +5206,7 @@
     "type": "terrain",
     "id": "t_tree_willow",
     "name": "willow tree",
+    "description":"A tall, wide deciduous tree of the 'Salix' genus, with long thin leafy branches that hang down like curtains.  You could tear off some bark if you examined the tree more closely.  You could cut it down with the right tools.",
```

Comment:
"if you examined the tree more closely" sounds a bit awkward, I'd just take it out...